Microsoft.Identity.Web/Client/TokenAcquisition.cs

```diff
@@ -125,9 +125,18 @@ public async Task AddAccountToCacheFromAuthorizationCode(AuthorizationCodeReceiv
             try
             {
                 // As AcquireTokenByAuthorizationCodeAsync is asynchronous we want to tell ASP.NET core that we are handing the code
-                // even if it's not done yet, so that it does not concurrently call the Token endpoint.
+                // even if it's not done yet, so that it does not concurrently call the Token endpoint. (otherwise there will be a
```

Comment:
this is just an improvement to the comment, as I got a customer question this week about this.